homeassistant/components/prometheus/__init__.py

```diff
@@ -491,18 +491,18 @@ def _handle_humidifier(self, state):
                     float(mode == current_mode)
                 )
 
-    def _handle_sensor(self, state):
+    def _handle_generic(self, domain, state):
         unit = self._unit_string(state.attributes.get(ATTR_UNIT_OF_MEASUREMENT))
 
-        for metric_handler in self._sensor_metric_handlers:
-            metric = metric_handler(state, unit)
+        for metric_handler in self._metric_name_handlers:
+            metric = metric_handler(state, unit, domain)
```

Comment:
May I please suggest to have `domain` as a first argument rather than the last? This will be consistent with `_handle_generic` and arguments will be sort of ordered from the most generic to the most specific.

Reply:
Done!